app/main.py

```diff
@@ -1 +1,50 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = list(set(self.skills))
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
```

Comment:
```suggestion
class SoftwareEngineer:
    def __init__(self, name: str) -> None:
        self.name = name
        self.skills = list(set(self.skills))

    def learn_skill(self, skill: str) -> None:
        self.skills.append(skill)
```

![image](https://github.com/user-attachments/assets/242a3223-c49b-4433-9802-e71ab2bbacc5)